automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeDataprocSpec.scala

```diff
@@ -0,0 +1,123 @@
+package org.broadinstitute.dsde.workbench.leonardo.runtimes
+
+import java.util.UUID
+
+import cats.effect.IO
+import cats.implicits._
+import cats.mtl.Ask
+import org.broadinstitute.dsde.workbench.google2.DataprocRole.{Master, SecondaryWorker, Worker}
+import org.broadinstitute.dsde.workbench.google2.{
+  DataprocClusterName,
+  GoogleDataprocInterpreter,
+  GoogleDataprocService,
+  MachineTypeName,
+  RegionName
+}
+import org.broadinstitute.dsde.workbench.leonardo.LeonardoApiClient.defaultCreateRuntime2Request
+import org.broadinstitute.dsde.workbench.leonardo.http.RuntimeConfigRequest
+import org.broadinstitute.dsde.workbench.leonardo.notebooks.{NotebookTestUtils, Python3}
+import org.broadinstitute.dsde.workbench.leonardo.{
+  ClusterCopy,
+  ContainerImage,
+  ContainerRegistry,
+  DataprocClusterStatus,
+  DiskSize,
+  GPAllocFixtureSpec,
+  LeonardoApiClient,
+  LeonardoConfig,
+  LeonardoTestUtils
+}
+import org.broadinstitute.dsde.workbench.model.TraceId
+import org.http4s.client.Client
+import org.http4s.headers.Authorization
+import org.http4s.{AuthScheme, Credentials}
+import org.scalatest.{DoNotDiscover, ParallelTestExecution}
+
+@DoNotDiscover
+class RuntimeDataprocSpec
```

Comment:
We don't really have a test currently that tests Dataproc functionality, so I added this one. It currently just has 1 test, to create a cluster with workers and preemptibles and verify all the nodes are created.